public/shared/messages.js

```diff
@@ -62,6 +62,10 @@ const SUCCESS_EXPORTING_DATASET_MESSAGE =
 const ERROR_EXPORTING_RESULT_MESSAGE =
   'There was an error exporting the result';
 const SUCCESS_EXPORTING_RESULT_MESSAGE = 'The result was exported successfully';
+const ERROR_EDITING_DATASET_FILE_MESSAGE =
+  'There was an error editing the dataset file';
+const ERROR_SETTING_DATASET_FILE_MESSAGE =
+  'There was an error editing the dataset';
```

Comment:
`ERROR_EDITING_DATASET_FILE_MESSAGE` is not used anywhere. I assume it got replaced by `ERROR_SETTING_DATASET_FILE_MESSAGE` and should be deleted